src/Engine/IO/Window.hpp

```diff
@@ -7,53 +7,58 @@
 #define GLEW_STATIC
 #include <GL/glew.h>
 #include <GLFW/glfw3.h>
+#include <string>
```

Comment:
Why do we need string here?

Reply:
string is used in the Window::init function

Reply:
as a parameter